cypress/support/step_definitions/validations.js

```diff
@@ -363,3 +363,34 @@ Given(/Interactions collection process is (initiated|stopped)/, (action) => {
 Given(/Validate Firebolt Interactions logs/, () => {
   cy.then(() => cy.validateFireboltInteractionLogs());
 });
+
+
+/**
+ * @module validations
+ * @function Given Validate Firebolt Interactions logs
+ * @description Validating the firebolt interaction logs in configModule
+ * @example
+ * Given Validate Firebolt Interactions logs
+ */
+Given(/'(.+)' on '(.+)' page/, (validationObjectKey, page) => {
+  if (Cypress.env('runtime')) {
+    Cypress.env('runtime').page = page;
+  } else {
+    Cypress.env('runtime', { page });
+  }
+
+  validationObjectKey = validationObjectKey.replaceAll(' ', '_').toUpperCase();
+  cy.getFireboltData(validationObjectKey).then((fireboltData) => {
+    const event = fireboltData.event;
```

Comment:
These lines below - could we just `resolveRecursiveValues` on the whole fireboltData object? And then pass that directly to `methodOrEventResponseValidation`? Not sure if we'd need to make some minor tweaks to the target in the object but that is how I was thinking it could work when I was writing it out.

Reply:
This is not a priority but just a thought to consider.

Reply:
Resolving entire fireboltData object and passing to `methodOrEventResponseValidation` will work when we created firebolt object with method, validationJsonPath, contentObject, eventExpected. Is it okay to change the firebolt object field names?

Reply:
Posted reply in the ticket.